src/test/java/hudson/plugins/copyartifact/DownstreamBuildSelectorTest.java

```diff
@@ -31,19 +31,10 @@
 import java.util.Map;
 import java.util.Set;
 
+import hudson.model.*;
 import hudson.plugins.copyartifact.testutils.CopyArtifactUtil;
 import jenkins.model.Jenkins;
 import hudson.FilePath;
-import hudson.model.Cause;
-import hudson.model.FreeStyleProject;
-import hudson.model.FreeStyleBuild;
-import hudson.model.Item;
-import hudson.model.ParametersAction;
-import hudson.model.ParametersDefinitionProperty;
-import hudson.model.StringParameterDefinition;
-import hudson.model.StringParameterValue;
-import hudson.model.User;
-import hudson.model.Result;

```

Comment:
I don't like wildcard import. (especially for classes)
Do you actually need this change?